examples/combined_ODNP.py

```diff
@@ -57,54 +50,101 @@
 myinput = input("Look ok?")
 if myinput.lower().startswith("n"):
     raise ValueError("you said no!!!")
-# }}}
-# {{{ these change if we change the way the data is saved
-IR_postproc = "spincore_IR_v1"
-Ep_postproc = "spincore_ODNP_v3"
-# }}}
 powers = 1e-3 * 10 ** (dB_settings / 10.0)
-fl = psp.figlist_var()
-save_file = True
-nPoints = int(acq_time_ms * SW_kHz + 0.5)
-acq_time_ms = nPoints / SW_kHz
-pad_us = 0
-Ep_ph1_cyc = psp.r_[0, 1, 2, 3]
-IR_ph1_cyc = psp.r_[0, 2]
-IR_ph2_cyc = psp.r_[0, 2]
-date = datetime.now().strftime("%y%m%d")
+# }}}
+#{{{phase cycling
+Ep_ph1_cyc = r_[0, 1, 2, 3]
+total_points = len(Ep_ph1_cyc)*nPoints
+assert total_points < 2**14, "For Ep: You are trying to acquire %d points (too many points) -- either change SW or acq time so nPoints x nPhaseSteps is less than 16384"%total_pts
+IR_ph1_cyc = r_[0, 2]
+IR_ph2_cyc = r_[0, 2]
+total_pts = len(IR_ph2_cyc)*len(IR_ph1_cyc)*nPoints
```

Comment:
even though these are default kwargs for the ppgs, the values are needed to calculate the total points to make sure we aren't acquiring too many points

Reply:
feed to ppg too